src/Resources/contao/languages/en/tl_content.php

```diff
@@ -17,6 +17,7 @@
 $GLOBALS['TL_LANG']['tl_content']['sliderButtons'] = ['Prev/next buttons','Shows the prev/next buttons.'];
 $GLOBALS['TL_LANG']['tl_content']['sliderPagination'] = ['Pagination','Shows a clickable pagination to navigate the slides.'];
 $GLOBALS['TL_LANG']['tl_content']['sliderPaginationType'] = ['Pagination type','The type of the pagination.'];
+$GLOBALS['TL_LANG']['tl_content']['sliderAutoheight'] = ['Autoheight','When active, the height of the slider adapts to the height of the currently active element.'];
```

Comment:
For the English translation I think it should be `Auto-height` (with a dash). But you could look at other Contao slider extensions - some of them also have an "auto height" option. I'd translate it the same as they do.

Reply:
I now use `Auto-Height`.
But I did not find another extension with auto-height.